src/frontend/utils/initWebinarContext.spec.ts

```diff
@@ -23,10 +23,13 @@ jest.mock('./checkLtiToken', () => ({
 
 jest.mock('./localstorage', () => ({
   getAnonymousId: jest.fn(),
+  setAnonymousId: jest.fn(),
 }));
 
+let mockDecodedJwtToken = {};
+
 jest.mock('data/appData', () => ({
-  getDecodedJwt: jest.fn(),
+  getDecodedJwt: () => mockDecodedJwtToken,
 }));
```

Comment:
I would suggest something like this

```suggestion
import { getDecodedJwt } from 'data/appData`;

 ...
  
jest.mock('data/appData', () => ({
  getDecodedJwt: jest.fn(),
}));

const mockGetDecodedJwt = getDecodedJwt as jest.MockedFunction<typeof getDecodedJwt>;
```

now `mockGetDecodedJwt` contains the `getDecodedJwt` mocked and in your test you can do something like `mockGetDecodedJwt.mockReturnValue(publicToken)`

This is the same mechanism used with `mockGetLiveRegistrations`

Reply:
yes true better to use the same method!